nupic/swarming/hypersearch/utils.py

```diff
@@ -753,3 +754,25 @@ def sortedJSONDumpS(obj):
 
   else:
     return json.dumps(obj)
+
+
+
+def matchPatterns(patterns, keys):
```

Comment:
I see it above, now.  I think it would be better if the `TODO` were closer to the actual function.